onnxruntime/python/tools/transformers/models/llama/benchmark.py

```diff
@@ -698,7 +691,8 @@ def main():
 
     # Measure prompt cost (init_inputs) and generated token cost (iter_inputs)
     for batch_size, sequence_length in itertools.product(args.batch_sizes, args.sequence_lengths):
-        logger.info(f"\nBatch size = {batch_size} and sequence length = {sequence_length}...")
+        if args.rank == 0:
+            logger.info(f"\nBatch size = {batch_size} and sequence length = {sequence_length}...")
```

Comment:
Same question as before for rank. The logger message is needed for the `benchmark_all.py` script.

Reply:
rank=0 means first gpu in current context. Say we run benchmark like CUDA_VISIBLE_DEVICES=3 python -m ...
here rank 0 means first gpu which is gpu:3 so logging in single gpu is not affected.
For multi-gpus case, we only want log from first gpu to reduce duplicates.